src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -7,190 +7,190 @@ partial class ApplicationEngine
     {
         public static readonly IReadOnlyDictionary<OpCode, long> OpCodePrices = new Dictionary<OpCode, long>
         {
-            [OpCode.PUSHINT8] = 30,
-            [OpCode.PUSHINT16] = 30,
-            [OpCode.PUSHINT32] = 30,
-            [OpCode.PUSHINT64] = 30,
-            [OpCode.PUSHINT128] = 120,
-            [OpCode.PUSHINT256] = 120,
-            [OpCode.PUSHA] = 120,
-            [OpCode.PUSHNULL] = 30,
-            [OpCode.PUSHDATA1] = 180,
-            [OpCode.PUSHDATA2] = 13000,
-            [OpCode.PUSHDATA4] = 110000,
-            [OpCode.PUSHM1] = 30,
-            [OpCode.PUSH0] = 30,
-            [OpCode.PUSH1] = 30,
-            [OpCode.PUSH2] = 30,
-            [OpCode.PUSH3] = 30,
-            [OpCode.PUSH4] = 30,
-            [OpCode.PUSH5] = 30,
-            [OpCode.PUSH6] = 30,
-            [OpCode.PUSH7] = 30,
-            [OpCode.PUSH8] = 30,
-            [OpCode.PUSH9] = 30,
-            [OpCode.PUSH10] = 30,
-            [OpCode.PUSH11] = 30,
-            [OpCode.PUSH12] = 30,
-            [OpCode.PUSH13] = 30,
-            [OpCode.PUSH14] = 30,
-            [OpCode.PUSH15] = 30,
-            [OpCode.PUSH16] = 30,
-            [OpCode.NOP] = 30,
-            [OpCode.JMP] = 70,
-            [OpCode.JMP_L] = 70,
-            [OpCode.JMPIF] = 70,
-            [OpCode.JMPIF_L] = 70,
-            [OpCode.JMPIFNOT] = 70,
-            [OpCode.JMPIFNOT_L] = 70,
-            [OpCode.JMPEQ] = 70,
-            [OpCode.JMPEQ_L] = 70,
-            [OpCode.JMPNE] = 70,
-            [OpCode.JMPNE_L] = 70,
-            [OpCode.JMPGT] = 70,
-            [OpCode.JMPGT_L] = 70,
-            [OpCode.JMPGE] = 70,
-            [OpCode.JMPGE_L] = 70,
-            [OpCode.JMPLT] = 70,
-            [OpCode.JMPLT_L] = 70,
-            [OpCode.JMPLE] = 70,
-            [OpCode.JMPLE_L] = 70,
-            [OpCode.CALL] = 22000,
-            [OpCode.CALL_L] = 22000,
-            [OpCode.CALLA] = 22000,
-            [OpCode.ABORT] = 30,
-            [OpCode.ASSERT] = 30,
-            [OpCode.THROW] = 22000,
-            [OpCode.TRY] = 100,
-            [OpCode.TRY_L] = 100,
-            [OpCode.ENDTRY] = 100,
-            [OpCode.ENDTRY_L] = 100,
+            [OpCode.PUSHINT8] = 4,
+            [OpCode.PUSHINT16] = 4,
+            [OpCode.PUSHINT32] = 4,
+            [OpCode.PUSHINT64] = 4,
```

Comment:
```suggestion
            [OpCode.PUSHINT8] = 1,
            [OpCode.PUSHINT16] = 1,
            [OpCode.PUSHINT32] = 1,
            [OpCode.PUSHINT64] = 1,
```

Please don't change the price in this PR.

Reply:
Well, we can concentrate on adjustability here, but we'll certainly need another PR for the fairness part of #1875. I think we have enough data now to prove that some prices are not representative of relative effort required to execute an instruction and this data also shows more fair values that could be used.

Fixed, though.